### ISSUE TITLE
Add 'symbol' command to tiflash

### ISSUE THREAD

**Issue by webbcam** [webbcam/tiflash#5]
Add command for getting/setting symbol value on device.

## Context

With the symbol command a user will have read/write access to the symbols in the device's image. This is similar to the expression command (and should use expression in its implementation) but will allow the user to load a symbol file for referencing the symbols directly by name.

This functionality is supported by the Debug Server Scripting API with the 'symbol' class. 
(See <CCS INSTALL DIR>/ti/ccsv8/ccs_base/scripting/docs/DS_API/index.html) Adding this to tiflash will just be a matter of providing an API for this functionality 

## Process

Below is the high-level process for adding this enhancement into TIFlash

1. Implement javascript side. Add 'symbol.js' which calls the Debug Server Script directly for running symbol commands (Follows structure of all other javascript modules: see erase.js for example)
2. Add symbol commands to TIFlash object in core/core.py
3. Add access to TIFlash symbol commands in core/api.py
4. Add symbol command arguments to core/args.py and update core/__main__.py with new args
5. Update all relevant __init__.py, docs, etc.

## Expected result

Symbol command should provide the following API:

**Getting Symbol Value:**
``` python
symbol.get(symbol_file, symbol_name):
    """Retrieves the value of the given symbol

    Args:
        symbol_file (str): full path to symbol image (.out)
        symbol_name (str): name of symbol to get value of

    Returns:
        str: returns the value of the symbol
    """
```

Setting Symbol Value:**
``` python
symbol.set(symbol_file, symbol_name, symbol_val):
    """Sets the value of the given symbol

    Args:
        symbol_file (str): full path to symbol image (.out)
        symbol_name (str): name of symbol to set value of
        symbol_val (str): value to set symbol too

    Returns:
        bool: returns True/False on success
    """
```

## Examples

CLI:
``` bash
$tiflash -s L4000CE symbol "/path/to/symbol.out" --get "symbol_name" 
"symbol_value"

$tiflash -s L4000CE symbol "/path/to/symbol.out" --set "symbol_name" "symbol_value"
True
```

Python:
``` python
from tiflash import core

core.symbol.get("/path/to/symbol.out", "symbol_name")    # returns "symbol_value"
core.symbol.set("/path/to/symbol.out", "symbol_name", "symbol_value") # returns True/False
```

----

**Comment by webbcam** [comment]
Will implement 'symbol_get' command first... 

'symbol_set' is going to be a bit trickier as it will require knowing what the value type is to set symbol to. (ie if I am setting 'symbolA' to true, tiflash needs to handle true as a boolean and not a string). 
Gonna kick this around a bit; may move to separate issue...

**Comment by webbcam** [comment]
All of this functionality is provided in the addition of the 'symbols' parameter to the "evaluate" command implemented here #12 
To read a symbol, simply provide the symbol file and the symbol name as the expression.
To write a symbol, provide the symbol file and the expression setting the symbol to a specific value.

Handling symbols this way is much easier and cleaner IMO as you don't run into the cases of variable type mentioned earlier.\

Closing ticket.
